assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,30 @@
+name: project_environment
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - cpuonly
+  - pip=20.2
+  - python=3.8.10
+  - pytorch=2.3.1
+  - torchvision=0.18.1
+  - pip:
+      - azureml-designer-cv-modules==0.0.59
+      - setuptools==65.5.1
```

Comment:
I don't think you need packages like wheel, setuptools explicitly referenced here. I believe that was done to mitigate vulnerabilities in the past, but with prioritized conda-forge channel, usually you will get the latest of those every time you materialize conda environment